plugins/net_plugin/net_plugin.cpp

```diff
@@ -321,12 +487,16 @@ namespace eosio {
    constexpr auto     def_max_trx_in_progress_size = 100*1024*1024; // 100 MB
    constexpr auto     def_max_consecutive_immediate_connection_close = 9; // back off if client keeps closing
    constexpr auto     def_max_clients = 25; // 0 for unlimited clients
+   constexpr auto     def_min_peers = 0;
    constexpr auto     def_max_nodes_per_host = 1;
    constexpr auto     def_conn_retry_wait = 30;
    constexpr auto     def_txn_expire_wait = std::chrono::seconds(3);
    constexpr auto     def_resp_expected_wait = std::chrono::seconds(5);
    constexpr auto     def_sync_fetch_span = 1000;
    constexpr auto     def_keepalive_interval = 10000;
+   constexpr auto     def_max_address_per_request = 10000;
+   constexpr auto     def_address_request_last_active_secs = std::chrono::seconds(600);
```

Comment:
10 minutes seems like a really long time for this. Seems like this should be more like 5 seconds.